eru/src/components/modules/cards/createScreelCard.vue

```diff
@@ -167,6 +191,30 @@ export default {
         });
       }
     },
+
+    async displayTagSuggestions(e) {
+      //allow only numbers and letters
+      if (!/[0-9a-zA-Z]/i.test(e.data)) {
+        e.preventDefault();
+        this.currentTag = this.currentTag.replace(e.data, "");
+        return;
+      }
+      if (this.currentTag.length === 0) return; //don't search for empty strings
+      let _vm = this;
+      debounce(async () => {
+        try {
+          const result = await axios.get(
```

Comment:
@Micheduc25 please can you do this in the store?

Reply:
I did not really think it was necessary to put it in the store because it will be used only in this page, and we are not accessing any value in the state of the application

Reply:
it's ok, leave it like that